### PR TITLE
Popup window implementation.

### PR DESCRIPTION
**Pull request by wale-prog**
This PR adds popup functionality to My portfolio project and also implements both the portfolio cards and the popup window in index.js to dynamically load in the browser when the page loads.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -15,3 +16,171 @@ navMenu.addEventListener('click', () => {
...
+    type
+    and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting
+    industry.
+    Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+    galley of type and scrambled it 1960s with the releax map lapora verita.`,
+    technologies: ['html', 'Bootstrap', 'Ruby on rails'],
+    liveLink: 'https://wale-prog.github.io/My-porfolio',
+    sourceLink: 'https://github.com/wale-prog/My-porfolio',
+  },
+];
```
Comment (vic778) [inline index.js:132]: kindly this window should contain `see live button` and `source code with a github logo` on the mobile device. add them to all windows for each project

**Review by vic778 — CHANGES_REQUESTED** [review]
## Hello @wale-prog 

Good job so far! I know you have worked hard on making this project.  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (wale-prog) [reply index.js:132]: Thank you for the feedback, I don't know how I missed that in the CSS file but it has been fixed now. Thanks.

**Review by zainsadaqat — APPROVED** [review]
# Hello @wale-prog  😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

## Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**